AlfrescoApp/View Controllers/Accounts View Controllers/New Account/AccountInfoViewController.h

```diff
@@ -55,6 +55,6 @@ typedef NS_ENUM(NSInteger, AccountActivityType)
  Only use this initialiser to init this view controller. Passing through nil for the account will display an empty controller
  which can be used for a new account setup.
  */
-- (id)initWithAccount:(UserAccount *)account accountActivityType:(AccountActivityType)activityType;
+- (id)initWithAccount:(UserAccount *)account accountActivityType:(AccountActivityType)activityType configuration:(NSDictionary *)configuration;
```

Comment:
It's often a good idea to add a second initializer rather than modifying an existing one - saves fallout from dependent code having to be edited.